rest_log/components/service.py

```diff
@@ -123,6 +123,15 @@ def _log_call_in_db_values(self, _request, *args, params=None, **kw):
             params = dict(params or {}, args=args)
 
         result = kw.get("result")
+        # NB: ``result`` might be an object of class ``odoo.http.Response``,
+        # for example when you try to download a file. In this case, we need to
+        # handle it properly, without the assumption that ``result`` is a dict.
+        if isinstance(result, Response):
+            status_code = result.status_code
+            result = {"status": status_code, "headers": dict(result.headers)}
```

Comment:
I think you must sanitize headers here like is done above. Maybe move it to a new method.

Can you also cherry-pick this fix? https://github.com/OCA/rest-framework/pull/301

Reply:
@simahawk done